dockstore-webservice/src/main/resources/migrations.1.16.0.xml

```diff
@@ -0,0 +1,175 @@
+                   context="1.15.0">
+    <changeSet author="ktran (generated)" id="addNumberOfSkippedExecutions">
+        <createSequence cacheSize="1" cycle="false" incrementBy="1" maxValue="9223372036854775807" minValue="1" sequenceName="metric_id_seq" startValue="1"/>
+        <sql dbms="postgresql">
+            <comment>Set the value of the metric_id_seq to the greatest value between statisticmetric_id_seq and countmetric_id_seq</comment>
+            SELECT setval('metric_id_seq', (SELECT last_value FROM statisticmetric_id_seq)) WHERE (SELECT last_value FROM metric_id_seq) &lt; (SELECT last_value FROM statisticmetric_id_seq);
+            SELECT setval('metric_id_seq', (SELECT last_value FROM countmetric_id_seq)) WHERE (SELECT last_value FROM metric_id_seq) &lt; (SELECT last_value FROM countmetric_id_seq);
+        </sql>
+        <createTable tableName="countmetric">
+            <column name="dbcreatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="dbupdatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="countmetric_pkey"/>
+            </column>
+        </createTable>
+        <createTable tableName="metric">
+            <column name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="metric_pkey"/>
+            </column>
+            <column name="numberofskippedexecutions" type="INTEGER"/>
+        </createTable>
+        <createTable tableName="statisticmetric">
+            <column name="average" type="FLOAT8">
+                <constraints nullable="false"/>
+            </column>
+            <column name="dbcreatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="dbupdatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="maximum" type="FLOAT8">
+                <constraints nullable="false"/>
+            </column>
+            <column name="minimum" type="FLOAT8">
+                <constraints nullable="false"/>
+            </column>
+            <column name="numberofdatapointsforaverage" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="unit" type="VARCHAR(255)"/>
+            <column name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="statisticmetric_pkey"/>
+            </column>
+        </createTable>
+        <sql dbms="postgresql">
+            INSERT INTO metric(id) SELECT id FROM validation_status;
+            INSERT INTO countmetric(id) SELECT id FROM validation_status;
+            INSERT INTO metric(id) SELECT id FROM execution_status;
+            INSERT INTO countmetric(id) SELECT id FROM execution_status;
+        </sql>
+        <!-- Migrate columns from cpu_metric to statisticmetric -->
```

Comment:
Thanks for powering through this!
I didn't quite realize the full  pain in migrating from the old schema to this one, but I'm hopeful it pays off.

Reply:
Should make a note that we should definitely use a full production dump when testing the 1.16.0 staging release